packages/faustwp-cli/utils/getWpUrl.ts

```diff
@@ -0,0 +1,14 @@
+import trim from 'lodash/trim.js';
+
+/**
+ * Returns the WordPress URL from the project's env.
+ */
+export function getWpUrl(path = ''): string {
+  const wpUrl = trim(process.env.NEXT_PUBLIC_WORDPRESS_URL, '/');
```

Comment:
This can get tricky because we have filters in the plugin system that users may rely on to modify their `wpUrl()` but they won't be applied here in the CLI.

Reply:
💯 agree that filters should work across all package (blocks, core & cli) and one that I would like to solve, however that issue existed prior to these changes, and think it should be dealt with in a separate PR.

The utility functions here were merely a means to centrally locate some repetitive code within the cli package.

Merging the core & cli packages could get us closer to that reality, but outside the scope of this PR.

Reply:
The challenge is that the introduction of this PR could cause a breaking change. Say for instance a user has their WPGraphQL endpoint as `/gql` instead of the default `/graphql`, and they are using the `graphqlEndpoint` Faust filter to modify that during runtime. With the introduction of this PR without the filter, their build would fail as the `/graphql` endpoint does not exist for them.

Reply:
@blakewilson 💯 - [this is already an issue with the current version](https://github.com/wpengine/faustjs/blob/canary/packages/faustwp-cli/utils/generatePossibleTypes.ts#L21). This PR is just centralizing that code.

Reply:
My concern is that while it is an issue in the current version, it is contained to just the `faust generatePossilbeTypes` script. So a user can still use Faust (they would have to generate types their own way if they stray from the default in the example project), but still usable. By making the changes in this PR, they are now blocked from even running `faust dev`, `faust build`, `faust start`, etc. And their Faust app breaks entirely until they adopt the `/graphql` endpoint naming convention.

Reply:
@theodesp @blakewilson Updated to throw a non-blocking warning message.